src/probnum/randprocs/covfuncs/_arithmetic_fallbacks.py

```diff
@@ -55,8 +55,8 @@ def __init__(self, covfunc: CovarianceFunction, scalar: ScalarLike):
     def _evaluate(self, x0: np.ndarray, x1: Optional[np.ndarray] = None) -> np.ndarray:
         return self._scalar * self._covfunc(x0, x1)
 
-    def _evaluate_linop(
-        self, x0: np.ndarray, x1: Optional[np.ndarray]
+    def linop(
+        self, x0: utils.ArrayLike, x1: Optional[utils.ArrayLike] = None
```

Comment:
Could you add `ArrayLike` to `from probnum.typing import [...]` above?

```suggestion
        self, x0: ArrayLike, x1: Optional[ArrayLike] = None
```

Reply:
Good catch!